script/Ciel/Day 11/11_01_CIEL11_1B.txt

```diff
@@ -102,20 +111,24 @@ This friend of mine goes back to his seat as soon as the bell signaling the star
 -- 「……………」
 "........."
 }
-[sha:8696f3bd82023f8ee0a176c16add6c4c9e99de64]{
+[offset:29118]{
 -- Page 44, Offset 29118.
 -- ……と。
-And then...
+// Death to ……と。
+However...
 }
 [sha:c724af6c70e35b4ba0eda6651c24d3d8a67c2584]{
 -- Page 44, Offset 29119.
 -- クラスメイトは上目遣いで俺の顔を見つめている。
-This classmate of mine glances up at me, staring at my face.
+This classmate of mine is still shyly staring at my face.
```

Comment:
I don't think it's trying to emphasize any difference between them, I think he's just ~morbing~ Roamoding out and describing them using relations instead of names.

Reply:
Yeah, that much I'm clear on, I was more thinking about the "this classmate of mine" and whatever in English. It seems sorta weirdly excessive? Normally it'd just be My friend/my classmate, but here both of them were stated like this and I didn't quite figure why.